src/ert/validation/runpath_argument.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from .argument_definition import ArgumentDefinition
+from .validation_status import ValidationStatus
+
+
+class RunPathArgument(ArgumentDefinition):
+    INVALID_PATH = "The specified runpath does not exist."
+    MISSING_PERMISSION = "You are missing permissions for the specified runpath."
+
+    def __init__(self, **kwargs: bool) -> None:
+        super().__init__(**kwargs)
+
+    def validate(self, token: str) -> ValidationStatus:
```

Comment:
Would it make sense to pass in token as a `Path` and not a string?

Reply:
It is a method we override from the super class `ArgumentDefinition`, so I don't think we can change it without changing all argument validations using the same super class.

Reply:
Ah I see, thanks.